packages/react/src/date-picker/docs/overview.mdx

```diff
@@ -23,14 +23,36 @@ figmaGalleryNodeId: 12444%3A100327
 
 ## Date picker
 
+`DatePicker` is a [controlled component](https://reactjs.org/docs/forms.html#controlled-components) which means consumers of this component need to manage the state of this component by using the `value` and `onChange` props.
```

Comment:
Made some nice documentation improvements while I was here adding content for the new props.